src/main/java/com/wojciechbarwinski/demo/epic_board_games_shop/EntityTest.java

```diff
@@ -0,0 +1,22 @@
+package com.wojciechbarwinski.demo.epic_board_games_shop;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
```

Comment:
Bezpieczniej jest dać nazwę tabelki, niż opierać się na springowo-hibernatowym mapowaniu nazwy pól

Reply:
Jasne, ale i tak zaraz przerzucamy się na LiquidBase. Tam wtedy damy nazwy a tu je zaktualizujemy pod te podane z LB. Teraz to po prostu encja czysto testowa